pkg/operator/ceph/csi/spec.go

```diff
@@ -349,11 +356,27 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 		if err != nil {
 			return errors.Wrap(err, "failed to load rbdplugin template")
 		}
+		if tp.CSILogRotation {
+			tp.CSILogFolder = "rbdDaemon"
```

Comment:
```suggestion
			tp.CSILogFolder = "rbdplugin"
```

Reply:
It should be the same as the container name as per @Madhu-1.

Reply:
we dont have any container with `rbdDaemon` @parth-gr where did you see one with this name in the csi pod?

Reply:
Ohh this is the folder name